src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -260,11 +256,31 @@ def get_pg_info(cls):
         }
 
 
+def get_rates_from_data(data):
```

Comment:
The code is duplicate in https://github.com/ceph/ceph/pull/28603/files#diff-d92e8344714a1099782f30b17c353d12.
Are there plans to refactor this after PR #28603 is merged?

Reply:
Hi, #28603 is a draft PR that was created because of @sebastian-philipp comment on this PR. In other words this PR should go in first. It's no duplication it will be a move one level up into the manager itself.